README.md

```diff
@@ -23,6 +23,9 @@ export APPNAME=<YOUR_HEROKU_APP_NAME>
 ```shell
 cd <HEROKU_PROJECT_ROOT_FOLDER>
 
+# Use the latest major Agent version
+export DD_AGENT_MAJOR_VERSION=7
```

Comment:
```suggestion
heroku config:add DD_AGENT_MAJOR_VERSION=7
```

Reply:
Could you explain why?

Reply:
Setting the environment variable locally (where you are running the `heroku` CLI, e.g. on a laptop) does not impact the environment variables inside the dyno where the agent runs. `heroku config` is how you set environment variables (such as `DD_API_KEY`) inside the dyno (see below in the readme).